tests/run.sh

```diff
@@ -4,8 +4,6 @@
 
 # To avoid permission denied error, please run `chmod +x tests/_utils/*`.
 
-set -e
```

Comment:
why

Reply:
Because we already have one on https://github.com/pingcap/dumpling/pull/305/files#diff-dd0c9cda3917fccebea6cea8d3fb46f46e4fbbeda2d6939864c9aaa9fc9c9571L16
Keep this line is also okay.